modules/sre-cluster-access.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa_policy_service_definition/rosa-sre-access.adoc
+
+:_content-type: CONCEPT
+[id="sre-cluster-access_{context}"]
+= SRE cluster access
+
+SRE access to {product-title} (ROSA) clusters can get full cluster administrative access through an escalation procedure; this procedure consists of several layers of authentication and multiple streams of approval, all of which are managed by strict company policy. All authentication attempts to access a cluster, as well as changes made within a ROSA cluster, are recorded within audit logs, along with the specific account identity of the SRE responsible for those actions. These audit logs help ensure that all changes made by SREs to a customer's ROSA cluster adhere to the strict policies and procedures that make up Red Hat's managed services guidelines.
+
+The information presented below is an overview of the process an SRE must perform to access a customer's ROSA cluster.
+
+** SRE makes a request to refresh ID token from the Red Hat SSO (Cloud Services).
+
+** SRE sends a request tunneled through the Red Hat VPN. This request is made via Corporate Identity and Access Management system (RH IAM); authentication is multi-factor (made up of a password and an ephemeral one-time token). Once the SRE authenticates and is allowed access to the orchestration and management systems, the authorization is managed by Red Hat corporate directory services. The use of RH IAM enables SREs to be managed internally per organization via groups and existing on-boarding/off-boarding processes. Changes to the orchestration and management systems require many layers of approval and are maintained by strict company policy.
+
+** Once authorized, SRE logs into the fleet management plane and receives a service account token that the fleet management plane created. The token is valid for twelve minutes. Once the token is no longer valid, it is deleted.
+
+** With access granted to the fleet management plane, the SRE will use various methods to access clusters, depending on network configuration.
+
+*** Accessing a private network cluster: Request is sent through a specific Network Load Balancer (NLB) using an encrypted HTTP connection. The NLB contains an IP allow-list so the APIs will accept connections from a specific set of IPs of which the fleet management plane contains.
+
+*** Accessing a PrivateLink cluster: Request is sent to the Red Hat Transit Gateway, which then connects to a Red Hat VPC per region. The VPC that receives the request will be dependent on the target private cluster’s region. Within the VPC there is a private subnet which contains the PrivateLink endpoint to the customer’s PrivateLink cluster.
```

Comment:
@jneczypor can chime in, but I think you should include a comma:

s/Within the VPC there is a private subnet/Within the VPC, there is a private subnet